sample-authz/src/main/java/net/trajano/ms/example/authz/sample/Clients.java

```diff
@@ -0,0 +1,25 @@
+package net.trajano.ms.example.authz.sample;
+
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.XmlSeeAlso;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'javax.xml.bind.annotation.XmlSeeAlso'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)